Presentation.Web/app/models/ViewModel/it-system/system-view-models.ts

```diff
@@ -0,0 +1,68 @@
+﻿module Kitos.Models.ViewModel.ItSystem {
+
+    export interface ISystemViewModel {
+        readonly name: string,
+        readonly parentName: string,
+        readonly previousName: string,
+        readonly belongsToName: string,
+        readonly accessModifier: string,
+        readonly description: string,
+        readonly externalReferences: any,
+        readonly taskRefs: any,
+        readonly businessTypeName: string,
+        readonly archiveDuty: string,
+        readonly uuid: string,
+    }
+
+    export class SystemViewModel implements ISystemViewModel {
+        readonly name: string;
+        readonly parentName: string;
+        readonly previousName: string;
+        readonly belongsToName: string;
+        readonly accessModifier: string;
+        readonly description: string;
+        readonly externalReferences: any;
+        readonly taskRefs: any;
+        readonly businessTypeName: string;
+        readonly archiveDuty: string;
+        readonly uuid: string;
+
+        constructor(itSystem: any) {
+            this.name = itSystem.name;
+            this.parentName = itSystem.parentName;
+            this.previousName = itSystem.previousName;
+            this.belongsToName = itSystem.belongsToName;
+            this.description = itSystem.description;
+            this.externalReferences = itSystem.externalReferences;
+            this.taskRefs = itSystem.taskRefs;
+            this.businessTypeName = itSystem.businessTypeName;
+            this.uuid = itSystem.uuid;
+
+            this.archiveDuty = this.mapArchiveDuty(itSystem.archiveDuty);
+            this.accessModifier = this.mapAccessModifier(itSystem.accessModifier);
+        }
+
+        private mapAccessModifier(accessModifier) {
+            switch (accessModifier) {
+                case 0:
+                    return "Lokal";
```

Comment:
Lige denne der er måske ret generel, så den kan vi med fordel mappe i en generisk mapper og ikke kun for itsystemusage. Accessmodifier anvendes jo andre steder, så på sigt kan vi få glæde af en generisk mapper.